data/governments.txt

```diff
@@ -1155,13 +1179,15 @@ government "Test Dummy"
 	"fine" 0
 
 # For space objects of unknown origin, to ensure the player can't land:
```

Comment:
This comment is for the "Unknown" gov, and so it should be moved to remain above it.